### DISCUSSION TITLE
部署时报错ImportError: cannot import name 'BaseOutputParser' from 'langchain.output_parsers'

### DISCUSSION THREAD

**Discussion by LibreJiayi** [GaiZhenbiao/ChuanhuChatGPT#315] [Q&A] [answered]
File "C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\site-packages\llama_index\output_parsers\langchain.py", line 6, in <module>
    from langchain.output_parsers import BaseOutputParser as LCOutputParser
ImportError: cannot import name 'BaseOutputParser' from 'langchain.output_parsers' (C:\Users\Administrator\AppData\Local\Programs\Python\Python311\Lib\site-packages\langchain\output_parsers\__init__.py)
已经确定正常安装，但是依然无法导入

**Comment by Keldos-Li** [comment, marked as answer]
> 把 `requirements.txt` 中 `langchain` 改为 `langchain<0.0.119` 重新安装依赖即可。[新版 langchain](https://github.com/hwchase17/langchain/commit/ce5d97bcb3e263f6aa69da6c334e35e20bf4db11) 修改了包结构

_Originally posted by @elonzh in https://github.com/GaiZhenbiao/ChuanhuChatGPT/issues/314#issuecomment-1480090662_

> ```shell
> pip install langchain==0.0.116
> pip install gradio==3.22.1
> ```

_Originally posted by @sunjianan9900 in https://github.com/GaiZhenbiao/ChuanhuChatGPT/issues/314#issuecomment-1480582130_
